synapse/logging/_terse_json.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Log formatters that output terse JSON.
+"""
+
+import sys
+from collections import deque
+from typing.io import TextIO
+
+import attr
+from simplejson import dumps
+
+from twisted.application.internet import ClientService
+from twisted.internet.endpoints import HostnameEndpoint
+from twisted.internet.protocol import Factory, Protocol
+from twisted.logger import FileLogObserver
+from twisted.python.failure import Failure
+
+
+def flatten_event(_event: dict, metadata: dict, include_time: bool = False):
```

Comment:
I find it odd to use `_event` tbh, it keeps making me think its unused. Can we call it `original_event` or name `event` `new_event` or something?

Reply:
done